packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -226,9 +226,9 @@ export const AutocompleteInput = <T extends AutocompleteInputOptionType>({
     onChange: handleFocusChange,
   })
 
-  const handleHide = useCallback(() => {
+  const handleHide = () => {
     onClose?.()
-  }, [onClose])
+  }
```

Comment:
This doesn't look like it actually closes the dropdown though? This is only conditionally calling the `onClose` callback; which is something that's passed in.

Isn't https://github.com/artsy/palette/blob/32df05b1e6d23dc9241bae775fd99527ebd869a3/packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx#L208-L210 what you want?

Reply:
I want the dropdown to be closed when the user clicks on the footer. 
Maybe we should call `resetUI()` instead of `onClose()`.

Reply:
```
   const handleClose = () => {
+    dispatch({ type: "CLOSE" })
+    reset()
     onClose?.()
   }
```

Consider the following: what do you actually want to happen? What action is in the footer? Do we want to close the drawer without resetting the value? Leave the value? I'd imagine the above is what you want.

Though I'm also a little confused by this PR, like the story didn't work to begin with?

I also think we should call this onClose not onHide, which is semantically closer to what's happening.